crates/utils/src/email.rs

```diff
@@ -19,6 +19,35 @@ pub mod translations {
 
 type AsyncSmtpTransport = lettre::AsyncSmtpTransport<lettre::Tokio1Executor>;
 
+#[async_trait::async_trait]
```

Comment:
None of this is necessary.

Reply:
Discussed in https://github.com/LemmyNet/lemmy/pull/3742#discussion_r1282285277